Dockerfile

```diff
@@ -12,12 +12,14 @@ RUN INSTALL_PKGS=" \
     yum clean all && \
     mkdir /tmp/ocp-clo && \
     chmod og+w /tmp/ocp-clo
-COPY --from=builder _output/bin/cluster-logging-operator /usr/bin/
+COPY --from=builder /go/src/github.com/openshift/cluster-logging-operator/_output/bin/cluster-logging-operator /usr/bin/
 COPY scripts/* /usr/bin/scripts/
 RUN mkdir -p /usr/share/logging/
 COPY files/ /usr/share/logging/
-COPY manifests/4* /manifests/
-COPY manifests/cluster-logging.package.yaml /manifests/
+
+COPY --from=builder /go/src/github.com/openshift/cluster-logging-operator/manifests /manifests
```

Comment:
Hard-coded path is a bit ugly, will break if the surrounding directory tree is ever moved. Not familiar with Dockerfiles, if there is  something like $PWD or $CURDIR or $(realpath ...) that works inside a Dockerfile that would be better - if not then may be the hard-coded path is the best we can do.

Reply:
You can define WORKDIR but this is the behavior difference between the previous build tool and this one; it is honored differently between layers.  Im less concerned as the likelyhood of this paths changes are small.